runtime/oti/util_api.h

```diff
@@ -47,6 +47,9 @@
 extern "C" {
 #endif
 
+/* Forward declaration. */
+struct jvmtiClassDefinition;
```

Comment:
Why did you decide to put it here rather than j9access.h?

Reply:
Every header file should be declare (or include) names it uses, and should not make assumptions about what has been declared before it is included.